utils/BetterFilterAPI.py

```diff
@@ -0,0 +1,219 @@
+from pathlib import Path
+import os
+
+SECTION_HEADER_FORMAT = "#==== AUTO GENERATED {} START ====\n"
+SECTION_FOOTER_FORMAT = "#==== AUTO GENERATED {} STOP ====\n"
+RULE_NAME_FORMAT = "# RULE {}\n"
+RULE_NAME_START = "# RULE"
+
+# A Section format is :
+"""
+#==== AUTO GENERATED <SECTION NAME> START ====
+# RULE <RULE1_NAME>
+rule
+contents
+blah
+...
+
+# RULE <RULE2_NAME>
+...
+#==== AUTO GENERATED <SECTION NAME> STOP ====
+"""
+
+
+def get_filter_directory():
+    """
+    Created by: 0xdavidel
+
+    Get the filter folder path
+    This path is always the filter directory
+    Proof - https://www.pathofexile.com/forum/view-thread/1835293/page/1#p14200591
```

Comment:
This post must be outdated. I've worked with multiple players who's filter paths are slightly different from expected due to non-english players. For example, a path would include `Dokuments` instead of the British English `Documents`.

My quick solution fore when this occurred would be to have the user interactively choose the correct folder via a GUI dialog.

Reply:
Not a problem to implement, Added to my TODO list.